src/notifications/notification.spec.ts

```diff
@@ -83,6 +83,58 @@ describe('`nglNotification`', () => {
     externalCloseButton.click();
   }));
 
+  it('should emit a close event when the 500ms timeout has passed', testAsync(({fixture, done}) => {
```

Comment:
No need to add testing specific numbers into the description. Just clarify the test expectation, ie: `should emit a close event when timeout duration has passed`.